doc/source/whatsnew/v0.25.0.rst

```diff
@@ -228,6 +228,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in :meth:`~pandas.eval` when comparing floats with scalar operators that don't have a 'value' attribute (e.g., unary operators) (:issue:`25928`)
```

Comment:
can you move down to numeric section. 

make this more like:

```
Bug in :meth:`~pandas.eval` when comparing floats with scalar operators, for example: ``x < 0.1`` (:issue:...)
```

Reply:
Yep, will do.